### PR TITLE
Init Project

### PR DESCRIPTION
**Pull request by mirulzaei**
ready

### CODE REVIEW

**Comment by mirulzaei** [comment]
ready

File: Gemfile [hunk 1]
```diff
@@ -0,0 +1,54 @@
+source 'https://rubygems.org'
+git_source(:github) { |repo| "https://github.com/#{repo}.git" }
+
+ruby '2.7.1'
+
+# Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
+gem 'rails', '~> 6.0.3', '>= 6.0.3.2'
+# Use sqlite3 as the database for Active Record
+gem 'sqlite3', '~> 1.4'
```
Comment (saobv-2223) [inline Gemfile]: image

File: app/javascript/packs/application.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+// This file is automatically compiled by Webpack, along with any other files
+// present in this directory. You're encouraged to place your actual application logic in
+// a relevant structure within app/javascript and only use these pack files to reference
+// that code so it'll be compiled.
+
+require("@rails/ujs").start()
+require("turbolinks").start()
+require("@rails/activestorage").start()
+require("channels")
```
Comment (saobv-2223) [inline app/javascript/packs/application.js]: image

File: app/mailers/application_mailer.rb [hunk 3]
```diff
@@ -0,0 +1,4 @@
+class ApplicationMailer < ActionMailer::Base
+  default from: 'from@example.com'
+  layout 'mailer'
```
Comment (saobv-2223) [inline app/mailers/application_mailer.rb]: image

File: app/views/layouts/application.html.erb [hunk 4]
```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>SampleApp</title>
+    <%= csrf_meta_tags %>
+    <%= csp_meta_tag %>
+
+    <%= stylesheet_link_tag 'application', media: 'all', 'data-turbolinks-track': 'reload' %>
+    <%= javascript_pack_tag 'application', 'data-turbolinks-track': 'reload' %>
```
Comment (saobv-2223) [inline app/views/layouts/application.html.erb]: image

File: config/credentials.yml.enc [hunk 5]
```diff
@@ -0,0 +1 @@
+GLjjARFFHAdzI3DOSsISmojY6DUqo3qOCmCpiQAO+pKbUS1Nkt4+R1kzC5d/T1Qq/eU1ErTC9PzhFV8WzIuqYW1qbyzJuNQVG8LFzFnj3Cicq49B59IODUMgxfzix7VyZPj/yggSnmN8RjUsE8sGv8LFNwf8/CjR3eUUQuiEBIEyAEJ4zuDJJWrJly6Mf5DE/2P97cf6dWyhIV2VzyXyTTzfKk3OdnGVc11cyOUS5Xl7O2QRKmHBBxQrIdDLtTuFlKPBvi/599GQPtGGfK1U7IUH4Fjt6alU5YJFb7zHUssUYp87ca/L7zGQuZtn2wv+ikjBO/1SfgmPwvP9oJTKG+DSIOyuFazVCb53xB5TV6lDuVrR+J9RNV05ZoadHPCYXVtg0ApicaHgl7zkSn+bTPF5AkpLrf6VInza--OPrfIpK91qUCUTaq--6uzycFrheMSOK2ojQrxcBg==
```
Comment (saobv-2223) [inline config/credentials.yml.enc]: image

File: config/database.yml [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+#
+#   Ensure the SQLite 3 gem is defined in your Gemfile
+#   gem 'sqlite3'
+#
+default: &default
+  adapter: sqlite3
+  pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
+  timeout: 5000
+
+development:
```
Comment (saobv-2223) [inline config/database.yml]: image

File: test/application_system_test_case.rb [hunk 7]
```diff
@@ -0,0 +1,5 @@
+require "test_helper"
+
+class ApplicationSystemTestCase < ActionDispatch::SystemTestCase
```
Comment (saobv-2223) [inline test/application_system_test_case.rb]: image

File: Gemfile (hunk 1 above)
Comment (mirulzaei) [reply Gemfile]: ready

**Comment by mirulzaei** [comment]
ready

File: Gemfile [hunk 8]
```diff
@@ -0,0 +1,59 @@
+source "https://rubygems.org"
+git_source(:github) { |repo| "https://github.com/#{repo}.git" }
+
+ruby "2.7.1"
+
+# Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
```
Comment (saobv-2223) [inline Gemfile]: remove all the comment lines

**Review by saobv-2223 — COMMENTED** [review]
fix the small mistake and merge

File: Gemfile (hunk 8 above)
Comment (mirulzaei) [reply Gemfile]: done pls check

**Comment by mirulzaei** [comment]
ready

File: Gemfile [hunk 9]
```diff
@@ -0,0 +1,55 @@
+source "https://rubygems.org"
+git_source(:github) { |repo| "https://github.com/#{repo}.git" }
+
+ruby "2.7.1"
+
```
Comment (chungpv-1008) [inline Gemfile:5]: please delete all blank lines from this line

File: config/application.rb [hunk 10]
```diff
@@ -0,0 +1,19 @@
+require_relative 'boot'
+
+require 'rails/all'
+
+# Require the gems listed in Gemfile, including any gems
```
Comment (chungpv-1008) [inline config/application.rb:5]: remove all comments in this file

File: config/routes.rb [hunk 11]
```diff
@@ -0,0 +1,3 @@
+Rails.application.routes.draw do
+  # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
```
Comment (chungpv-1008) [inline config/routes.rb:2]: remove all comments

**Review by chungpv-1008 — COMMENTED** [review]
merged

**Pull request merged**